bench-exchange/src/bench.rs

```diff
@@ -11,7 +11,7 @@ use solana_drone::drone::request_airdrop_transaction;
 use solana_exchange_api::exchange_instruction;
 use solana_exchange_api::exchange_state::*;
 use solana_exchange_api::id;
-use solana_metrics::influxdb;
+use solana_metrics::*;
```

Comment:
Can we explicitly bring the new macros into the namespace rather than using a wildcard?  Future code readers will otherwise not know where the macros are coming from.  Or call them with solana_metrics::submit!() (or whatever the corresponding macro syntax is)